examples/resources/hw_documentation/resource.tf

```diff
@@ -0,0 +1,24 @@
+terraform {
```

Comment:
Why `hw_documentation` ?  It's not specific to Hardware

Reply:
This is addressed in the latest commit. Thanks